index.js

```diff
@@ -15,17 +15,71 @@
  */
 
 var _fs = require("fs");
+var net = require("net");
 var http = require("http");
+var https = require("https");
 var URL = require("url");
 var PATH = require("path");
 var mime = require("mime");
 var heuristic = require("./heuristic");
 
 exports = module.exports = serverReplay;
 function serverReplay(har, options) {
-    var server = http.createServer(makeRequestListener(har.log.entries, options));
+    var fs = options.fs || _fs;
+    if (!options.ssl) {
+        options.ssl = {
+            key: __dirname + "/ssl/snakeoil.key",
+            cert: __dirname + "/ssl/snakeoil.crt"
```

Comment:
Could you use `path.join` here. I haven't tested on Windows, but nice to keep compat if it does work